src/actions/actions.js

```diff
@@ -0,0 +1,5 @@
+const firstMessage = {
+  type: 'FIRST_MESSAGE',
```

Comment:
In my opinion, I don't see why this file should be called actions.js, since you are puting in there types, you should have called it types.js because actions doesn't describe the real content of it, 

we can call actions, a file which contains some functions which perform some actions such as retrieving data or sending data and dispatching state

Reply:
> In my opinion, I don't see why this file should be called actions.js, since you are puting in there types, you should have called it types.js because actions doesn't describe the real content of it,
> 
> we can call actions, a file which contains some functions which perform some actions such as retrieving data or sending data and dispatching state

it's not a type, it's an action that only has a type, I followed the way I've seen on Redux website, check `https://redux.js.org/basics/actions`.
the file that performs those actions is called reducer because they are the ones which receive actions and state.

Reply:
I have just read on that link, and I think it will be more clear as we keep on moving.